src/uss/ussNodeActions.ts

```diff
@@ -70,4 +70,38 @@ export async function deleteFromDisk(node: ZoweUSSNode, filePath: string) {
             }
         }
         catch (err) {}
+}
+
+export async function initializeUSSFavorites(ussFileProvider: USSTree) {
+    const lines: string[] = vscode.workspace.getConfiguration("Zowe-USS-Persistent-Favorites").get("favorites");
+    lines.forEach(async line => {
+        const profileName = line.substring(1, line.lastIndexOf("]"));
+        const nodeName = (line.substring(line.indexOf(":") + 1, line.indexOf("{"))).trim();
+        const session = await utils.getSession(profileName);
+        let node: ZoweUSSNode;
+        if (line.substring(line.indexOf("{") + 1, line.lastIndexOf("}")) === "directory") {
+        node = new ZoweUSSNode(
+            nodeName,
+            vscode.TreeItemCollapsibleState.Collapsed,
+            ussFileProvider.mFavoriteSession,
+            session,
+            "",
+            false,
+            profileName
+        );
+        } else {
+            node = new ZoweUSSNode(
+                nodeName,
+                vscode.TreeItemCollapsibleState.None,
```

Comment:
You could delete this whole else statement and just:
```suggestion
                nodeType,
```
nodeType would be defined above.

Reply:
Great suggestion! As we discussed, we decided to leave it as because the `else` is necessary for the `node.command`